dist/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>Webpack Starter Kit</title>
+  </head>
+  <body>
+    <article class='full-panel'>
+      <h1>Webpack Starter Kit</h1>
+      <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    </article>
+    <article class="login-panel">
```

Comment:
Just a heads up, this might work better inside of a `form` element - also check the `type` of the input for your password and username fields, there is some default form behavior that is helpful. Lastly, your submit button might need to be an `input` as well - at least check those things if you have strange behavior in your code!